.github/CONTRIBUTING.md

```diff
@@ -371,13 +371,103 @@ This does not go for bug reports however, as we want to unearth and solve as man
 
 ### Commit
 
-* Focus on one thing per commit 
-* Keep changes as small as possible 
-* Only commit related functinoality for each commit 
-* Commit often
-* Strive for each commit to compile/build and pass tests 
-* Use [gitmoji](https://gitmoji.dev) when possible - point to VSCode extension and CLI tool
-* sidenote: when merging the CI pipeline combines each commit to a merge commit containing all commit messages
+You may have encountered statements like
+
+> Commit early, Commit often
+
+We agree. Apart from acting as checkpoints, good commits support collaboration by providing an overview of your changes and make navigating them easy. But only as long the commits are good.
+
+#### The good commit follows Git best practices
+
+You are encouraged to follow commonly adopted Git best practices. We would especially like to highlight:
+
+1. **Try to make each commit a logically separate changeset**
+    - Make changes in each commit _atomic_ &ndash; focus on one thing
+    - Keep your commits as small as possible, but keep your changes in each commit related
+    - If you have made multiple non-related changes and need to split them into separate commits you can use `git add --patch` to stage them
+2. **Commit often**
+    - It is easier to get an overview with multiple small changes than one large commit containing lots of changes
+    - If you need to roll back to a previous commit, you can do it incrementally instead of reverting every change
+3. **Strive for each commit to compile/build and pass tests**
+4. **Regularly push your commits to remote**
+    - Early peer reviews are easier
+    - In case something happens to your machine it reduces the amount of work lost
+    - Makes it easier to hand over the issue to other developers if needed
+
+#### The good commit has good commit messages
+
+> Keep in mind: [This](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) &middot; [has](https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines) &middot; [all](https://github.com/torvalds/subsurface-for-dirk/blob/master/README.md#contributing) &middot; [been](http://who-t.blogspot.co.at/2009/12/on-commit-messages.html) &middot; [said](https://github.com/erlang/otp/wiki/writing-good-commit-messages) &middot; [before](https://github.com/spring-projects/spring-framework/blob/30bce7/CONTRIBUTING.md#format-commit-messages)
+>
+> _&mdash; Chris Beams, [The seven rules of a great Git commit message](https://chris.beams.io/posts/git-commit/#seven-rules)_
+
+All commits from your branch are squashed when merged with the main branch. Commit messages from individual commits end up in a list of changes in the message body of the merge commit. Try to keep that in mind when writing your commit messages.
+
+Ideally, a good commit message will be structured into three parts:
+
+1. Subject
+2. Body
```

Comment:
To match below sentence: "The message body should be used if elaboration is necessary."
```suggestion
2. Message body
```